src/sentry/static/sentry/app/components/dataExport.tsx

```diff
@@ -26,11 +31,6 @@ type State = {
   dataExportId?: number;
 };
 
-const TooltipMessages = {
-  start: "We'll get all your data in one place and email you when it's ready",
-  progress: "We'll email you when it's ready",
-} as const;
```

Comment:
I removed this object because it really wasn't reused anywhere, and seemed kind of unnecessary